bindings/python/manifold3d.cpp

```diff
@@ -315,6 +315,12 @@ NB_MODULE(manifold3d, m) {
       .def("calculate_curvature", &Manifold::CalculateCurvature,
            nb::arg("gaussian_idx"), nb::arg("mean_idx"),
            manifold__calculate_curvature__gaussian_idx__mean_idx)
+      .def("calculate_normals", &Manifold::CalculateNormals,
+           nb::arg("normal_idx"), nb::arg("min_sharp_angle"),
+           manifold__calculate_normals__normal_idx__min_sharp_angle)
+      .def("smooth", &Manifold::Smooth, nb::arg("min_sharp_angle"),
```

Comment:
@wrongbad I appear to have messed up these python bindings somehow or maybe just the docs? Any thoughts on what I did wrong?